### PR TITLE
Fix column breaks on Firefox

### PR DESCRIPTION
**Pull request by A1Asriel**
## Description

This PR adds a simple (and kinda stupid) solution to the longstanding column break issue on Firefox. Since it doesn't support the native `column-break-after` CSS property, this adds a 100vh bottom margin to `.columnSplit` elements and fill all available space to push everything below to the next column. Empty `@-moz-document url-prefix()` at-rule prevents Chromium and other browsers from enabling this hack.

It doesn't seem to break anything - both web version and print preview look fine and similar to the Chromium rendering (except for font rendering differences).

## Related Issues or Discussions

Solves (couldn't find an open issue for this bug):
- #543
- #2473
- #3712

## QA Instructions, Screenshots, Recordings

*Current VS fixed version on Firefox Nightly 149.0a1 (2026-01-31)*
firefox

*Current VS fixed version on Chromium 146.0.7663.0 (Developer Build)*
chrome

Note: the only change I made to the document in the fixed version was replacing Imgur links with another host as it was throwing 403 in the locally hosted app for some reason.

### Reviewer Checklist

*Reviewers, refer to this list when testing features, or suggest new items *
- [ ] Verify new features are functional
  - [x] Column breaks are working on Firefox
- [ ] Verify old features have not broken
  - [ ] Page layout is not affected on Chromium
  - [ ] No side effects arise on Firefox
- [ ] Test for edge cases / try to break things
- [ ] Identify opportunities for simplification and refactoring
- [ ] Check for code legibility and appropriate comments

### CODE REVIEW

**Comment by Gazook89** [comment]
Sorry, I can't pull it down right now, but does it continue to work when a PDF is generated?

File: client/homebrew/brewRenderer/brewRenderer.less [hunk 1]
```diff
@@ -59,6 +59,12 @@
 		}
 		&-corner { visibility : hidden; }
 	}
+
+	@-moz-document url-prefix() {
+		.columnSplit {
+			margin-bottom: 100vh;
+		}
+	}
```
Comment (5e-Cleric) [inline client/homebrew/brewRenderer/brewRenderer.less:67]: We shall not use the non-standard and deprecated syntax `moz-document`, instead the supports at-rule is made for this use.
```suggestion

	@supports (break-after:always) {
		.columnSplit {
			margin-bottom: 100vh;
		}
	}
```

Comment (dbolack-ab) [reply client/homebrew/brewRenderer/brewRenderer.less:67]: Should this be a `@supports not (break-after: always) {` ?

Comment (5e-Cleric) [reply client/homebrew/brewRenderer/brewRenderer.less:67]: That would make more sense, but apparently chrome only supports `-webkit-column-break-after`, for some reason, and firefox supports `break-after` only, (still ignores the hell out of it tho)

Comment (dbolack-ab) [reply client/homebrew/brewRenderer/brewRenderer.less:67]: No, it should not. :)

Comment (dbolack-ab) [reply client/homebrew/brewRenderer/brewRenderer.less:67]: Both the original and suggested change work delightfully well for me locally.

Comment (A1Asriel) [reply client/homebrew/brewRenderer/brewRenderer.less:67]: > We shall not use the non-standard and deprecated syntax `moz-document`, instead the supports at-rule is made for this use.

Huh, I didn't know about this rule, but I suppose it fits better! Testing it showed the exact same results as well.

**Comment by A1Asriel** [comment]
> Sorry, I can't pull it down right now, but does it continue to work when a PDF is generated?

Yes it does. The left one is generated via Firefox and the right one - through Chromium, for reference.

**Comment by 5e-Cleric** [comment]
Simple to review and test, merging now, thanks for the contribution @A1Asriel
